### PR TITLE
List past Astropy awards received

### LINKED ISSUE #500: New page to list awards received
Example: [2022 ADASS Software Prize](https://astropy.slack.com/archives/C067V74GK/p1660867645252449)

@adrn thinks this can be done by staffing https://github.com/astropy/astropy-project/pull/284

### PR DESCRIPTION
**Pull request by pllim**
Fix #500 

Screenshot 2022-11-10 141232

### CODE REVIEW

**Comment by jdswinbank** [comment]
By my reading, both of those awards are to the team, not to the software. That's a slightly obtuse distinction, but I think it does matter: the awards are not directly an endorsement of Astropy as a piece of software, but a recognition of the work the team has put in. It would be possible (arguably, it happens all the time) for a prizeworthy team to create lousy software, and (less likely) for prizeworthy software to be created by a lousy team. 

As such, I reckon these should be listed on `team.html` rather than `about.html`.

File: about.html [hunk 1]
```diff
@@ -178,6 +178,19 @@ <h1 id="fiscal-sponsor">Fiscal Sponsor<a class="paralink" href="#fiscal-sponsor"
...
 	</section>
 
+    <section>
+        <a name="awards"></a>
+
+        <h1 id="awards">Awards<a class="paralink" href="#awards" title="Permalink to this headline">¶</a></h1>
+
+        <p>
+            <ul>
+                <li/><a href="https://www.adass.org/softwareprize.html">ADASS Software Prize</a> (2022)
```
Comment (jdswinbank) [inline about.html:188]:
```suggestion
                <li/><a href="https://www.adass.org/softwareprize.html">ADASS Prize for an Outstanding Contribution to Astronomical Software</a> (2022)
 ```

File: about.html [hunk 2]
```diff
@@ -178,6 +178,19 @@ <h1 id="fiscal-sponsor">Fiscal Sponsor<a class="paralink" href="#fiscal-sponsor"
...
+                <li/><a href="https://ras.ac.uk/awards-and-grants/awards/2269-group-award-a">RAS Group Award</a> (2020)
```
Comment (jdswinbank) [inline about.html:189]:
```suggestion
                <li/><a href="https://ras.ac.uk/awards-and-grants/awards/2269-group-award-a">Royal Astronomical Society Group Award (Astronomy)</a> (2020)
 ```

**Comment by pllim** [comment]
Thanks, @jdswinbank ! I moved it.

**Review by hamogu — APPROVED** [review]
It's been discussed, everyone on the PR agrees, let's merge it!

**Pull request merged**